src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Reader/CborReader.cs

```diff
@@ -81,6 +81,35 @@ public ReadOnlyMemory<byte> ReadEncodedValue(bool disableConformanceModeChecks =
             return _data.Slice(initialOffset, _offset - initialOffset);
         }
 
+        /// <summary>
+        /// Resets the <see cref="CborReader"/> instance over the specified <paramref name="data"/> with unchanged configuration.
+        /// <see cref="ConformanceMode"/> and <see cref="AllowMultipleRootLevelValues"/> are unchanged.
+        /// </summary>
+        /// <param name="data">The CBOR-encoded data to read.</param>
+        public void Reset(ReadOnlyMemory<byte> data)
+        {
+            // ConformanceMode = conformanceMode;
+            // AllowMultipleRootLevelValues = allowMultipleRootLevelValues;
```

Comment:
These comments are very vague, we should improve them.

```suggestion
            // ConformanceMode and AllowMultipleRootLevelValues are set in ctor, they remain unchanged.
```